src/Data/List/Relation/Unary/Any.agda

```diff
@@ -67,9 +67,8 @@ infixr 5 _∷=_
 _∷=_ : {P : Pred A p} → Any P xs → A → List A
 _∷=_ {xs = xs} x∈xs v = xs List.[ index x∈xs ]∷= v
 
-infixl 4 _─_
-_─_ : {P : Pred A p} → ∀ xs → Any P xs → List A
-xs ─ x∈xs = xs List.─ index x∈xs
```

Comment:
This is a slightly different operation, it doesn't take an index, so I don't think `removeAt` is maybe the best name for this one.

I think for the sake of not letting this PR grow out of control, let's keep this function as is in this PR!

Reply:
Yes: to keeping it out of the PR :+1:
No: (at least with my dependently typed spectacles on) it *does* take an (abstract!) 'index', `x∈xs`, but one which is more interestingly typed, as `Any P xs`. 
But let's take it off the table for the time being ;-)